preboots/nsq.js

```diff
@@ -38,7 +38,12 @@ module.exports = function nsqboot(app, options, callback) {
 
   app.nsq.handler = new StatusHandler({
     conc: app.config.get('nsq:concurrency'),
-    models: app.models
+    models: app.models,
+    webhooks: {
+      concurrency: app.config.get('webhooks:concurrency'),
+      endpoints: app.webhookEndpoints,
+      timeout: app.config.get('webhooks:concurrency')
```

Comment:
```suggestion
      timeout: app.config.get('webhooks:timeout')
```

Reply:
And I’d probably do the `app.config.get` lines next to eachother visually.

Reply:
LOL "Visually"

Reply:
I like better the fact that the params are ordered alphabetically. But ok since you're the maintainer here I am happy to change it :)